query/src/storages/fuse/operations/gc.rs

```diff
@@ -29,7 +30,15 @@ use crate::storages::fuse::FuseTable;
 
 impl FuseTable {
     pub async fn do_gc(&self, ctx: &Arc<QueryContext>, keep_last_snapshot: bool) -> Result<()> {
-        let snapshot_opt = self.read_table_snapshot(ctx.as_ref()).await?;
+        let r = self.read_table_snapshot(ctx.as_ref()).await;
+        let snapshot_opt = match r {
+            Err(e) if e.code() == ErrorCode::storage_not_found_code() => {
+                // concurrent gc: someone else has already collected this snapshot, ignore it
```

Comment:
We can have a warning log here to know this is a concurrent gc.

Reply:
> Can we also create a transient fuse table with the options with the old planner?


> Can we also create a transient fuse table with the options with the old planner?

yeah. both planners are all supported

https://github.com/datafuselabs/databend/blob/ac3d9554c6f1a208450b930c9743c906b7cf9fa8/tests/suites/0_stateless/05_ddl/05_0000_ddl_create_tables.sql#L78

https://github.com/datafuselabs/databend/blob/ac3d9554c6f1a208450b930c9743c906b7cf9fa8/tests/suites/0_stateless/05_ddl/05_0000_ddl_create_tables_v2.sql#L78